INSTALL

```diff
@@ -27,11 +27,13 @@ WHAT DO YOU NEED ?
 
      port install coq
 
-   To compile Coq V8.8 yourself, you need:
+   To compile Coq yourself, you need:
 
    - OCaml version 4.02.3 or later
      (available at https://ocaml.org/)
 
+   - The Num package, which used to be part of the OCaml standard library.
```

Comment:
You could maybe add that this changed for users >= 4.06

Reply:
One thing I'd like to understand better is: is the num package needed only for version >= 4.06 or also for earlier versions so that it's properly registered with ocamlfind? (cf.#7678)

Reply:
This seems to be distro specific AFAICT, by default in OPAM for versions less that 4.06.0 you will have:
```
num                 (version: [distributed with Ocaml])
```